src/functionalTest/resources/features/F-002 DELETE Hearing/S-002.1-get-hearing.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "_guid_": "S-002.1-get-hearing",
+  "_extends_" : "GetHearingRequestPositiveGeneric",
+  "specs": [
+    "to verify versionNumber=2 and status=CANCELLATION_REQUESTED"
+  ],
+  "request": {
+    "pathVariables": {
+      "hearingID" : "${[scenarioContext][siblingContexts][CreateHearingRequest][testData][actualResponse][body][hearingRequestID]}"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 200,
```

Comment:
"_extends_": "Common_200_Response",

Reply:
Done in [2188106](https://github.com/hmcts/hmc-cft-hearing-service/pull/219/commits/218810605b79e9828fd4211d2f41169c7ce26897)